DQMOffline/Trigger/python/TopMonitoring_cff.py

```diff
@@ -1,4 +1,369 @@
+fullyhadronic_DoubleBTag_all.denGenericTriggerEventPSet.hltPaths = cms.vstring('HLT_IsoMu27_v')
+
+fullyhadronic_DoubleBTag_jet = hltTOPmonitoring.clone()
+fullyhadronic_DoubleBTag_jet.FolderName   = cms.string('HLT/TopHLTOffline/TopMonitor/FullyHadronic/DoubleBTag/JetMonitor/')
+# Selections
+fullyhadronic_DoubleBTag_jet.leptJetDeltaRmin = cms.double(0.0)
+fullyhadronic_DoubleBTag_jet.njets            = cms.uint32(6)
+fullyhadronic_DoubleBTag_jet.jetSelection     = cms.string('pt>30 & abs(eta)<2.4')
+fullyhadronic_DoubleBTag_jet.HTdefinition     = cms.string('pt>30 & abs(eta)<2.4')
+fullyhadronic_DoubleBTag_jet.HTcut            = cms.double(450)
+fullyhadronic_DoubleBTag_jet.nbjets           = cms.uint32(2)
+fullyhadronic_DoubleBTag_jet.bjetSelection    = cms.string('pt>30 & abs(eta)<2.4')
+fullyhadronic_DoubleBTag_jet.workingpoint = cms.double(0.8484) # Medium
+# Binning 
+fullyhadronic_DoubleBTag_jet.histoPSet.htPSet = cms.PSet(nbins=cms.int32(50), xmin=cms.double(0.0), xmax=cms.double(1000) )
+fullyhadronic_DoubleBTag_jet.histoPSet.jetPtBinning = cms.vdouble(0,5,10,15,20,25,30,35,40,45,50,55,60,65,70,75,80,90,100,120,200,400)
+fullyhadronic_DoubleBTag_jet.histoPSet.HTBinning    = cms.vdouble(0,420,440,460,480,500,520,540,560,580,600,650,700,750,800,850,900)
+# Triggers
+fullyhadronic_DoubleBTag_jet.numGenericTriggerEventPSet.hltPaths = cms.vstring('HLT_PFHT380_SixJet32_DoubleBTagCSV_p075_v')
+fullyhadronic_DoubleBTag_jet.denGenericTriggerEventPSet.hltPaths = cms.vstring('HLT_PFHT370_v')
+
+fullyhadronic_DoubleBTag_bjet = hltTOPmonitoring.clone()
+fullyhadronic_DoubleBTag_bjet.FolderName   = cms.string('HLT/TopHLTOffline/TopMonitor/FullyHadronic/DoubleBTag/BJetMonitor/')
+# Selections
+fullyhadronic_DoubleBTag_bjet.leptJetDeltaRmin = cms.double(0.0)
+fullyhadronic_DoubleBTag_bjet.njets            = cms.uint32(6)
+fullyhadronic_DoubleBTag_bjet.jetSelection     = cms.string('pt>30 & abs(eta)<2.4')
+fullyhadronic_DoubleBTag_bjet.HTdefinition     = cms.string('pt>30 & abs(eta)<2.4')
+fullyhadronic_DoubleBTag_bjet.HTcut            = cms.double(450)
+fullyhadronic_DoubleBTag_bjet.nbjets           = cms.uint32(2)
+fullyhadronic_DoubleBTag_bjet.bjetSelection    = cms.string('pt>30 & abs(eta)<2.4')
+fullyhadronic_DoubleBTag_bjet.workingpoint     = cms.double(0.70)
+# Binning
+fullyhadronic_DoubleBTag_bjet.histoPSet.htPSet = cms.PSet(nbins=cms.int32(50), xmin=cms.double(0.0), xmax=cms.double(1000) )
+fullyhadronic_DoubleBTag_bjet.histoPSet.jetPtBinning = cms.vdouble(0,5,10,15,20,25,30,35,40,45,50,55,60,65,70,75,80,90,100,120,200,400)
+fullyhadronic_DoubleBTag_bjet.histoPSet.HTBinning    = cms.vdouble(0,420,440,460,480,500,520,540,560,580,600,650,700,750,800,850,900)
+# Triggers
+fullyhadronic_DoubleBTag_bjet.numGenericTriggerEventPSet.hltPaths = cms.vstring('HLT_PFHT380_SixJet32_DoubleBTagCSV_p075_v')
+fullyhadronic_DoubleBTag_bjet.denGenericTriggerEventPSet.hltPaths = cms.vstring('HLT_PFHT380_SixJet32_v')
```

Comment:
Hi Mia, 

For the monitoring of the full paths we use the HLT_IsoMu27 path as a reference and the SingleMuon dataset.

Now, for the legs of the paths we use the JetHT dataset:

For the Jet Leg:
Numerator:     HLT_PFHT380_SixJet32  
Denominator: HLT_PFHT370

and 

Numerator:     HLT_PFHT430_SixJet40  
Denominator:  HLT_PFHT430 


For the B-Jet Leg:

Numerator:      HLT_PFHT380_SixJet32_DoubleBTagCSV_p075
Denominator:  HLT_PFHT380_SixJet32

and

Numerator:    HLT_PFHT430_SixJet40_BTagCSV_p080
Denominator: HLT_PFHT430_SixJet40


For all cases plots of the jets pt, eta, phi, ht, bjet pt, eta, phi, csv are created. 

Any suggestion is welcome :)